plugins/poll/lib/polls_validator.rb

```diff
@@ -21,6 +23,32 @@ def validate_polls
 
     private
 
+    def valid_arguments?(poll)
+      valid = true
+
+      unless [nil, "regular", "multiple", "number"].include?(poll["type"])
```

Comment:
We try to avoid duplicating hardcoded values (and creating unnecessary arrays). What about this instead?

```ruby
if poll["type"].present? && !Poll.types.has_key?(poll["type"])
```